packages/outline-button/src/outline-button.css

```diff
@@ -22,8 +22,8 @@
   align-items: center;
   cursor: pointer;
   transition-property: color, background-color, border-color,
-    text-decoration-color, fill, stroke, outline, outline-color;
-  transition-timing-function: cubic-bezier(0.4, 0, 0.2, 1);
+    text-decoration-color, fill, stroke, outline, outline-color, box-shadow;
+  transition-timing-function: ease-in;
```

Comment:
In order to generalize this enough for Outline, these three CSS rules will need to have CSS Variables in places matching the pattern of other in the component that allow for them to be customized, and turned off by default.